botmother/management/commands/polling.py

```diff
@@ -0,0 +1,28 @@
+import json
+import time
+from django.conf import settings
+from django.core.management.base import BaseCommand
+
+from botmother.utils.api import TelegramAPI
+from botmother.utils.bot_router import BotRouter
+from example.urls import dispatch
+
+
+class Command(BaseCommand):
+    def handle(self, *args, **kwargs):
+        telegram = TelegramAPI(settings.BOT_TOKEN)
+        updates_id = []
```

Comment:
What is the reason of collecting update ids?

Reply:
Because, bot every 5 seconds sends all older messages. I think it doesn't work like that. Bot sends every new messages. **updates_id** is older updates id, I'm checking for old messages.